doc/wg/core/notes/core-notes-2021-07-02.md

```diff
@@ -0,0 +1,333 @@
+# Tock Core Notes 2021-07-02
+
+Attending:
+- Alexandru Radovici
+- Amit Levy
+- Anthony Quiroga
+- Arjun Deopujari
+- Brad Campbell
+- Hudson Ayers
+- Johnathan Van Why
+- Leon Schuermann
+- Pat Pannuto
+- Philip Levis
+
+## Updates
+* Johnathan: Unfortunately this only applies to RISC-V, at least with the
+  current plans. On behalf of OpenTitan, LowRISC is working on an embedded
+  position-independent-code spec for RISC-V that they're contributing to LLVM
+  for both `clang` and `rustc`.
+* Amit: That's awesome. Do we know if it's going to support the kind of
+  PIC we rely on right now in ARM for C?
+* Johnathan: The current plans are such that it will initially look a lot like
+  that, but then it will change to be somewhat more efficient and eliminate the
+  global offset table. It was specifically designed for Tock's usage.
+* Amit: That would be huge, that's great.
```

Comment:
This is incredibly exciting. All the more so if it's being designed for OpenTitan explicitly. My memory from when we were first exploring this way back when is that ARM PIC does a bunch of little inefficient things that are unnecessary from a Tock perspective. The GOT being the biggest example.